burger2/style/_gallery.scss

```diff
@@ -0,0 +1,46 @@
+h2 {
```

Comment:
you write now in every file the styles which apply to EVERY page

so don't assign styles to tags globally without wrapping to some class